code/modules/reagents/reagents/drugs.dm

```diff
@@ -220,3 +220,25 @@
 	M.stats.addTempStat(STAT_TGH, -STAT_LEVEL_BASIC, STIM_TIME, "sanguinum_w")
 	M.stats.addTempStat(STAT_COG, -STAT_LEVEL_BASIC, STIM_TIME, "sanguinum_w")
 	M.stats.addTempStat(STAT_ROB, -STAT_LEVEL_BASIC, STIM_TIME, "sanguinum_w")
+
+
+/datum/reagent/drug/mindwipe
+	name = "Mindwipe"
+	id = "mindwipe"
+	description = "Shocks the user's brain hard enough to make him forget about his quirks. Is ill-advised because of side effects"
+	taste_description = "bitter"
+	reagent_state = LIQUID
+	color = "#bfff00"
+	metabolism = REM
+	overdose = REAGENTS_OVERDOSE * 3
+	nerve_system_accumulations = 95
```

Comment:
So, this is to just instantly breach nerve system accumulation?

Reply:
Yup. This drug should kick you really hard, it shouldn't be a "I don't want to play with sanity mechanics" pill.

Reply:
But 95 will not breach it. Clearly value of 110 or higher needed for this drug to be as strong as it should.